.github/ISSUE_TEMPLATE/02_feature_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Feature Request
+description: Enchancement to admiral functionality
+title: "Feature Request:  <Insert Issue Title Here>"
+labels: ["enhancement", "programming"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!  We love keeping admiral fresh!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: feature
+    attributes:
+      label: Feature Idea (Succint)?
+      description: Tell us your idea in as few words as possible
+      placeholder: Tell us what you see!
+      value: "`derive_var_dtm` should do x, y and z "
+    validations:
+      required: true
+  - type: textarea
+    id: output
+    attributes:
+      label: Relevant Output
+      description: Can you provide what the final output should look like?
+      placeholder: Tell us!
+      value: "What should the output look like?"
+    validations:
+      required: false
+  - type: textarea
+    id: code
+    attributes:
+      label: Reproducible Example/Pseudo Code
+      description: Can you provide a working example or a sketch of how the code should work?
+      placeholder: Tell us see!
```

Comment:
Typo: I assume it should be "Tell us!".

Reply:
So this doesn't show up in the template when it renders, but I fixed it.  You can see how they render by clicking on the files 
![image](https://user-images.githubusercontent.com/10111024/152048816-ce0af6d5-dab9-498b-900f-06a341388e64.png)